docs/content/direct/commission-workloads.md

```diff
@@ -0,0 +1,5 @@
+# Deploying Workloads KubeStellar Usage Examples
+
+We are still working on generalized step-by-step instructions for defining, binding and deploying particular workloads.
```

Comment:
I think that the terminology here still needs some work.

I make a distinction between "workload objects" and "control objects". I think that this file is trying to talk about creating the desired state for both sorts of object. The reported state for each does _not_ come from the user.